src/Services/Gateways/BaseGateway.php

```diff
@@ -10,11 +10,30 @@
 
 abstract class BaseGateway extends HttpService
 {
+    const COUNTRY_BRAZIL   = 'Brazil';
```

Comment:
There's a model called Country which has the constants already defined there on [line 7](https://github.com/ebanx/benjamin/blob/master/src/Models/Country.php#L7).